pkg/kubeapiserver/options/authentication.go

```diff
@@ -223,6 +225,10 @@ func (s *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 			"Comma-separated list of allowed JOSE asymmetric signing algorithms. JWTs with a "+
 			"'alg' header value not in this list will be rejected. "+
 			"Values are defined by RFC 7518 https://tools.ietf.org/html/rfc7518#section-3.1.")
+
+		fs.Var(flag.NewMapStringString(&s.OIDC.RequiredClaims), "oidc-required-claims", ""+
```

Comment:
will the claim values ever have commas? if they are something like distinguished names, they could. if so, we'd need a different flag type that you repeat rather than split on `,`

Reply:
Glancing through the [standard claims](http://openid.net/specs/openid-connect-core-1_0.html#Claims) it looks like the claim values could possibly have commas (e.g the address claim). I will go ahead and write up a new flag type.